sphinx/source/docs/dev_guide/testing.rst

```diff
@@ -12,11 +12,11 @@ be viewed at: https://travis-ci.org/bokeh/bokeh.
 
 To run the just the python unit tests, run the command::
 
-    $ python -c "import bokeh; bokeh.test()"
+    python -c "import bokeh; bokeh.test()"
```

Comment:
While we're editing this, might as well add the `.. code-block:: python` above (and `sh` for the other lines in this file).